### PR TITLE
streamgo.to

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/119613

popunder

### CODE REVIEW

File: EnglishFilter/sections/general_extensions.txt [hunk 1]
```diff
@@ -282,6 +282,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/119613
+vikistream.com#%#//scriptlet("remove-class", "clickable", ".player-poster")
```
[the remark this answers is not in the document]
Comment (jellizaveta) [reply EnglishFilter/sections/general_extensions.txt:286]: Yesterday was available, was working.

[the remark this answers is not in the document]
Comment (zloyden) [reply EnglishFilter/sections/general_extensions.txt:286]: How does it work today? I haven't seen any popups.

[the remark this answers is not in the document]
Comment (jellizaveta) [reply EnglishFilter/sections/general_extensions.txt:286]: Yes, checked it. The rule works for me.

**Review by zloyden — DISMISSED** [review]

**Comment by AdamWr** [comment]
>the rule does not work on iOS

The same on my device.

Could you please add all these rules:
```
cdn1.link###adholder
vikistream.com,embedstream.me,streamgo.to#%#//scriptlet("prevent-window-open")
streamgo.to#%#//scriptlet("abort-on-property-write", "_pop")
vikistream.com,streamgo.to#%#//scriptlet("abort-current-inline-script", "document.createElement", "delete window")
vikistream.com$$script[wildcard="*function*break;case*"][min-length="30000"][max-length="100000"]
```
These rules seem to work fine for me.

**Comment by zloyden** [comment]
> The same on my device.

I can confirm that the rules work fine on my end.
@jellizaveta do not add this on iOS :)
vikistream.com$$script[wildcard="*function*break;case*"][min-length="30000"][max-length="100000"]

**Comment by Yuki2718** [comment]
EL has generic `###adholder`.

**Comment by Yuki2718** [comment]
`vikistream.com,streamgo.to#%#//scriptlet("abort-current-inline-script", "document.createElement", "delete window")`
Of note I see propeller delete-window staff only in `vikistream.com` and not in `streamgo.to`.

**Comment by AdamWr** [comment]
It seems that I was wrong and `cdn1.link###adholder` also doesn't work.
This one works fine:
```
cdn1.link$$div[id="adholder"]
```

<hr>

> `vikistream.com,streamgo.to#%#//scriptlet("abort-current-inline-script", "document.createElement", "delete window")`
> Of note I see propeller delete-window staff only in `vikistream.com` and not in `streamgo.to`.

I know, but iOS is problematic and as I recall there was issue (or it was by design) that rules were not applied to iframes, so domain from "top" document have to be used.

**Comment by AdamWr** [comment]
@jellizaveta Could you please add these rules:
```
cdn1.link$$div[id="adholder"]
vikistream.com,embedstream.me,streamgo.to#%#//scriptlet("prevent-window-open")
vikistream.com,streamgo.to#%#//scriptlet("abort-current-inline-script", "document.createElement", "delete window")
vikistream.com$$script[wildcard="*function*break;case*"][min-length="30000"][max-length="100000"]
```

**Comment by Yuki2718** [comment]
I'll directly add some revolving adservers, but pls check on your end too @jellizaveta as they change frequently.

**Review by zloyden — APPROVED** [review]

**Review by AdamWr — APPROVED** [review]

**Pull request merged**